packages/app/src/paths.ts

```diff
@@ -18,3 +18,7 @@ export function getPathForPlatform (posixPath?: string) {
 
   return posixPath
 }
+
+export function posixify (path?: string): string {

```

Comment:
Can this be undefined? That seems weird.

Reply:
some of the places where we call this function pass things in from GraphQL query results where they might not be defined. I'll update this to pass `''` at the call sights instead